rom/laghos_rom.cpp

```diff
@@ -471,7 +479,7 @@ ROM_Basis::ROM_Basis(ROM_Options const& input, MPI_Comm comm_, MPI_Comm rom_com_
 
     if (offsetInit)
     {
-        std::string path_init = basename + "/ROMoffset" + input.basisIdentifier + "/init";
+        std::string path_init = basename + "/ROMoffset" + input.basisIdentifier + "/init"; // TODO: think about parametric case
```

Comment:
Is this still an issue? If so, what remains to be resolved?

Reply:
This is now fixed.